common/authenticator-requests/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "nym-authenticator-requests"
+version = "0.1.0"
+authors.workspace = true
+repository.workspace = true
+homepage.workspace = true
+documentation.workspace = true
+edition.workspace = true
+license.workspace = true
+
+[dependencies]
+bincode = { workspace = true }
+rand = "0.8.5"
```

Comment:
Use workspace dependency?

Reply:
Fixed it here and in `ip-packet-requests` too